src/main/java/uk/gov/hmcts/reform/civil/service/DashboardNotificationsParamsMapper.java

```diff
@@ -22,6 +24,14 @@ public Map<String, Object> mapCaseDataToParams(CaseData caseData) {
         params.put("defaultRespondTime", "4pm");
         params.put("defendantName", caseData.getRespondent1().getPartyName());
 
+        if (nonNull(getDefendantAdmittedAmount(caseData))) {
+            params.put("defendantAdmittedAmount", formatAmount(getDefendantAdmittedAmount(caseData)));
+        }
+
+        if (nonNull(caseData.getRespondToClaimAdmitPartLRspec())) {
+            params.put("whenWillThisAmountBePaid", caseData.getRespondToClaimAdmitPartLRspec().getWhenWillThisAmountBePaid());
```

Comment:
could the name of the parameter "whenWillThisAmountBePaid" be more specific?

Reply:
Fixed, I change it to the name that you suggested me: "defendantAdmittedAmountPaymentDeadline"